src/features/form/containers/GroupContainer.tsx

```diff
@@ -233,58 +237,65 @@ export function GroupContainer({ id, container, components }: IGroupProps): JSX.
-      {editIndex >= 0 && edit?.mode === 'hideTable' && (
-        <RepeatingGroupsEditContainer
-          container={container}
-          editIndex={editIndex}
-          setEditIndex={setEditIndex}
-          repeatingGroupIndex={repeatingGroupIndex}
-          id={id}
-          language={language}
-          textResources={textResources}
-          layout={layout}
-          repeatingGroupDeepCopyComponents={repeatingGroupDeepCopyComponents}
-          multiPageIndex={multiPageIndex}
-          setMultiPageIndex={setMultiPageIndex}
-          filteredIndexes={filteredIndexList}
-        />
-      )}
-      {edit?.mode === 'showAll' &&
-        // Generate array of length repeatingGroupIndex and iterate over indexes
-        Array(repeatingGroupIndex + 1)
-          .fill(0)
-          .map((_, index) => {
-            if (filteredIndexList && filteredIndexList.length > 0 && !filteredIndexList.includes(index)) {
-              return null;
-            }
+      <ConditionalWrapper
+        condition={!isNested}
+        wrapper={(children) => <FullWidthWrapper>{children}</FullWidthWrapper>}
+      >
+        <>
+          {editIndex >= 0 && edit?.mode === 'hideTable' && (
+            <RepeatingGroupsEditContainer
+              container={container}
+              editIndex={editIndex}
+              setEditIndex={setEditIndex}
+              repeatingGroupIndex={repeatingGroupIndex}
+              id={id}
+              language={language}
+              textResources={textResources}
+              layout={layout}
+              repeatingGroupDeepCopyComponents={repeatingGroupDeepCopyComponents}
+              multiPageIndex={multiPageIndex}
+              setMultiPageIndex={setMultiPageIndex}
+              filteredIndexes={filteredIndexList}
+            />
+          )}
+          {edit?.mode === 'showAll' &&
+            // Generate array of length repeatingGroupIndex and iterate over indexes
+            Array(repeatingGroupIndex + 1)
+              .fill(0)
+              .map((_, index) => {
+                if (filteredIndexList && filteredIndexList.length > 0 && !filteredIndexList.includes(index)) {
+                  return null;
+                }
 
-            return (
-              <RepeatingGroupsEditContainer
-                key={index}
-                editIndex={index}
-                repeatingGroupIndex={repeatingGroupIndex}
-                container={container}
-                id={id}
-                language={language}
-                deleting={deletingIndexes.includes(index)}
-                textResources={textResources}
-                layout={layout}
-                setEditIndex={setEditIndex}
-                onClickRemove={onClickRemove}
-                repeatingGroupDeepCopyComponents={repeatingGroupDeepCopyComponents}
-                forceHideSaveButton={true}
-              />
-            );
-          })}
+                return (
+                  <div
+                    key={index}
```

Comment:
I see you did not change this, but i think the id is safer to use as key here than the index.
```suggestion
                    key={`key-${id}`}
```

Reply:
This is not code that is touched by this PR, so it's out of scope. If this need to be changed it could be made in another issue.